src/pages/Embeds/Plio.vue

```diff
@@ -807,16 +795,21 @@ export default {
         this.lastTimestampRetention = currentTime;
       }
     },
-    // checks if the scorecard needs to pop up or not
+    /** checks if the scorecard needs to pop up or not */
     checkForScorecardPopup(timestamp) {
-      if (timestamp >= this.player.duration - 1 && this.hasAnyItems) {
+      if (
+        timestamp >= this.player.duration - SCORECARD_POPUP_TIME_FROM_END &&
+        this.hasAnyItems
```

Comment:
I think it might be helpful for readability and for future as well to have a function - `isScorecardEnabled` - which can be used here as well as here 👇 
![Screenshot 2021-09-27 at 12 00 52 PM](https://user-images.githubusercontent.com/13185699/134855782-9bdf58df-5204-4672-b73f-e9c734f26c03.png)

Reply:
Also, the check for whether the scorecard is enabled or not should ideally happen where the `checkForScorecardPopup` function is being called - if not enabled, don't call this function. It probably shouldn't be the duty of the function to check if the scorecard is enabled or not.